diff --git a/vrecord b/vrecord
index 7353211..8faf95d 100755
--- a/vrecord
+++ b/vrecord
@@ -33,7 +33,7 @@ FFPLAY_DECKLINK="${BREW_PREFIX}/bin/ffplay-dl"
 FFPROBE_DECKLINK="${BREW_PREFIX}/bin/ffprobe-dl"
 MPVOPTS=(--no-osc)
 MPVOPTS+=(--load-scripts=no)
-MPVOPTS+=(--script "${RESOURCE_PATH}/qcview.lua")
+MPVOPTS+=(--script="${RESOURCE_PATH}/qcview.lua")
 MPVOPTS+=(--really-quiet)
 
 if [[ "$("${FFMPEG_DECKLINK}" -version 2>&1 | grep "Library not loaded" >/dev/null)" || ! -f "${FFMPEG_DECKLINK}" || ! -f "${FFPLAY_DECKLINK}" ]] ; then
