diff --git a/vrecord b/vrecord
index 7353211..094966a 100755
--- a/vrecord
+++ b/vrecord
@@ -485,7 +485,7 @@ _home_gui(){
                 <button>
                     <label>Documentation</label>
                     <input file>${RESOURCE_PATH}/vrecord_logo_documentation.png</input>
-                    <action>open https://github.com/amiaopensource/vrecord#vrecord-documentation</action>
+                    <action>xdg-open https://github.com/amiaopensource/vrecord#vrecord-documentation</action>
                 </button>
             </hbox>
         </vbox>
